deployment/ccip/changeset/initial_add_chain.go

```diff
@@ -0,0 +1,32 @@
+package changeset
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+)
+
+var _ deployment.ChangeSet[DeployCCIPContractConfig] = InitialAddChain
```

Comment:
definitely should rename this config struct - NewChainsConfig? Also lets keep the definition of the config here